### DISCUSSION TITLE
2 OSCP boards on a website

### DISCUSSION THREAD

**Discussion by meganjoell** [bullhorn/career-portal#458] [General]
My boss is planning on expanding our business and I am creating the new website for our company. We staff in advertising/marketing and would like to start a legal side of our business. I am curious to know you can have 2 OSCPs on the same website (would be located in different pages). I am not skill in web development so if this is completely out of pocket to ask I apologize.

**Comment by charlesabarnes** [comment]
Yes. You just need to set them up in separate directories in your web host

**Reply by meganjoell** [reply]
Thank you for your quick response, I will look into that (hopefully without breaking our website )
